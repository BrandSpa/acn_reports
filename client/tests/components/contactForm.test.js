import React from 'react';
import { shallow, mount } from 'enzyme';
import moxios from 'moxios';
import ContactForm from '../../components/contactForm';

describe('component ContactForm', () => {
  beforeEach(() => {
    moxios.install();
  });

  afterEach(() => {
    moxios.uninstall();
  });

  it('should render as expected', () => {
    const wrapper = shallow(<ContactForm />);
    expect(wrapper.getNodes()).toMatchSnapshot();
  });

  it('should have 3 inputs and 1 select', () => {
    const wrapper = shallow(<ContactForm />);
    expect(wrapper.find('input').length).toBe(3);
    expect(wrapper.find('select').length).toBe(1);
  });

  it('should change country on componentDidMount', () => {
    const wrapper = mount(<ContactForm country="Germany" countries={['Argentina', 'Colombia']} />);
    expect(wrapper.props().country).toBe('Germany');
  });

  it('should change contact state', () => {
    const name = 'ale';
    const lastname = 'san';
    const email = 'alesan@gmail.com';
    const e = {};
    const wrapper = shallow(<ContactForm />);
    wrapper
      .find('input')
      .at(0)
      .simulate('change', { e, target: { value: name } });

    wrapper
      .find('input')
      .at(1)
      .simulate('change', { e, target: { value: lastname } });

    wrapper
      .find('input')
      .at(2)
      .simulate('change', { e, target: { value: email } });

    expect(wrapper.state().contact).toEqual({
      name,
      lastname,
      email,
      country: '',
    });
  });

  it('should show validations', () => {
    const wrapper = shallow(<ContactForm />);
    wrapper.find('button').simulate('click', { preventDefault: () => {} });
    expect(wrapper.find('.input-error').length).toBe(3);
  });
});
