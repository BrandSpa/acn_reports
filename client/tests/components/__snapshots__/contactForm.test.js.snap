// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component ContactForm should render as expected 1`] = `
Array [
  <form
    className="form-inline contact-form"
    data-jsx={2272587868}
    onSubmit={[Function]}
    style={
        Object {
            "textAlign": "center",
          }
    }
>
    <div
        className="input-container"
        data-jsx={2272587868}
        style={
            Object {
                "@media (maxWidth: 767px)": Object {
                  "width": "100%",
                },
                "width": "20%",
              }
        }
    >
        <input
            data-jsx={2272587868}
            onChange={[Function]}
            placeholder={undefined}
            style={
                Object {
                    "borderRadius": "",
                  }
            }
            type="text"
            value=""
        />
        <div
            className="hidden"
            data-jsx={2272587868}
        >
             
        </div>
    </div>
    <div
        className="input-container"
        data-jsx={2272587868}
        style={
            Object {
                "@media (maxWidth: 767px)": Object {
                  "width": "100%",
                },
                "width": "20%",
              }
        }
    >
        <input
            data-jsx={2272587868}
            onChange={[Function]}
            placeholder={undefined}
            style={
                Object {
                    "borderRadius": "",
                  }
            }
            type="text"
            value=""
        />
        <div
            className="hidden"
            data-jsx={2272587868}
        />
    </div>
    <div
        className="input-container"
        data-jsx={2272587868}
        style={
            Object {
                "@media (maxWidth: 767px)": Object {
                  "width": "100%",
                },
                "width": "20%",
              }
        }
    >
        <input
            data-jsx={2272587868}
            onChange={[Function]}
            placeholder={undefined}
            style={
                Object {
                    "borderRadius": "",
                  }
            }
            type="text"
            value=""
        />
        <div
            className="hidden"
            data-jsx={2272587868}
        />
    </div>
    <div
        className="input-container"
        data-jsx={2272587868}
        style={
            Object {
                "@media (maxWidth: 767px)": Object {
                  "width": "100%",
                },
                "width": "20%",
              }
        }
    >
        <select
            data-jsx={2272587868}
            onChange={[Function]}
            value=""
        >
            <option
                data-jsx={2272587868}
                value=""
            />
            <option
                data-jsx={2272587868}
                value="Afghanistan"
            >
                Afghanistan
            </option>
            <option
                data-jsx={2272587868}
                value="Albania"
            >
                Albania
            </option>
            <option
                data-jsx={2272587868}
                value="Algeria"
            >
                Algeria
            </option>
            <option
                data-jsx={2272587868}
                value="Andorra"
            >
                Andorra
            </option>
            <option
                data-jsx={2272587868}
                value="Angola"
            >
                Angola
            </option>
            <option
                data-jsx={2272587868}
                value="Antigua and Barbuda"
            >
                Antigua and Barbuda
            </option>
            <option
                data-jsx={2272587868}
                value="Argentina"
            >
                Argentina
            </option>
            <option
                data-jsx={2272587868}
                value="Armenia"
            >
                Armenia
            </option>
            <option
                data-jsx={2272587868}
                value="Australia"
            >
                Australia
            </option>
            <option
                data-jsx={2272587868}
                value="Austria"
            >
                Austria
            </option>
            <option
                data-jsx={2272587868}
                value="Azerbaijan"
            >
                Azerbaijan
            </option>
            <option
                data-jsx={2272587868}
                value="Bahamas"
            >
                Bahamas
            </option>
            <option
                data-jsx={2272587868}
                value="Bahrain"
            >
                Bahrain
            </option>
            <option
                data-jsx={2272587868}
                value="Bangladesh"
            >
                Bangladesh
            </option>
            <option
                data-jsx={2272587868}
                value="Barbados"
            >
                Barbados
            </option>
            <option
                data-jsx={2272587868}
                value="Belarus"
            >
                Belarus
            </option>
            <option
                data-jsx={2272587868}
                value="Belgium"
            >
                Belgium
            </option>
            <option
                data-jsx={2272587868}
                value="Belize"
            >
                Belize
            </option>
            <option
                data-jsx={2272587868}
                value="Benin"
            >
                Benin
            </option>
            <option
                data-jsx={2272587868}
                value="Bhutan"
            >
                Bhutan
            </option>
            <option
                data-jsx={2272587868}
                value="Bolivia"
            >
                Bolivia
            </option>
            <option
                data-jsx={2272587868}
                value="Bosnia and Herzegovina"
            >
                Bosnia and Herzegovina
            </option>
            <option
                data-jsx={2272587868}
                value="Botswana"
            >
                Botswana
            </option>
            <option
                data-jsx={2272587868}
                value="Brazil"
            >
                Brazil
            </option>
            <option
                data-jsx={2272587868}
                value="Brunei"
            >
                Brunei
            </option>
            <option
                data-jsx={2272587868}
                value="Bulgaria"
            >
                Bulgaria
            </option>
            <option
                data-jsx={2272587868}
                value="Burkina Faso"
            >
                Burkina Faso
            </option>
            <option
                data-jsx={2272587868}
                value="Burundi"
            >
                Burundi
            </option>
            <option
                data-jsx={2272587868}
                value="Cambodia"
            >
                Cambodia
            </option>
            <option
                data-jsx={2272587868}
                value="Cameroon"
            >
                Cameroon
            </option>
            <option
                data-jsx={2272587868}
                value="Canada"
            >
                Canada
            </option>
            <option
                data-jsx={2272587868}
                value="Cape Verde"
            >
                Cape Verde
            </option>
            <option
                data-jsx={2272587868}
                value="Central African Republic"
            >
                Central African Republic
            </option>
            <option
                data-jsx={2272587868}
                value="Chad"
            >
                Chad
            </option>
            <option
                data-jsx={2272587868}
                value="Chile"
            >
                Chile
            </option>
            <option
                data-jsx={2272587868}
                value="China"
            >
                China
            </option>
            <option
                data-jsx={2272587868}
                value="Colombia"
            >
                Colombia
            </option>
            <option
                data-jsx={2272587868}
                value="Comoros"
            >
                Comoros
            </option>
            <option
                data-jsx={2272587868}
                value="Congo"
            >
                Congo
            </option>
            <option
                data-jsx={2272587868}
                value="Costa Rica"
            >
                Costa Rica
            </option>
            <option
                data-jsx={2272587868}
                value="Croatia"
            >
                Croatia
            </option>
            <option
                data-jsx={2272587868}
                value="Cuba"
            >
                Cuba
            </option>
            <option
                data-jsx={2272587868}
                value="Cyprus"
            >
                Cyprus
            </option>
            <option
                data-jsx={2272587868}
                value="Czech Republic"
            >
                Czech Republic
            </option>
            <option
                data-jsx={2272587868}
                value="Denmark"
            >
                Denmark
            </option>
            <option
                data-jsx={2272587868}
                value="Djibouti"
            >
                Djibouti
            </option>
            <option
                data-jsx={2272587868}
                value="Dominica"
            >
                Dominica
            </option>
            <option
                data-jsx={2272587868}
                value="Dominican Republic"
            >
                Dominican Republic
            </option>
            <option
                data-jsx={2272587868}
                value="East Timor"
            >
                East Timor
            </option>
            <option
                data-jsx={2272587868}
                value="Ecuador"
            >
                Ecuador
            </option>
            <option
                data-jsx={2272587868}
                value="Egypt"
            >
                Egypt
            </option>
            <option
                data-jsx={2272587868}
                value="El Salvador"
            >
                El Salvador
            </option>
            <option
                data-jsx={2272587868}
                value="Equatorial Guinea"
            >
                Equatorial Guinea
            </option>
            <option
                data-jsx={2272587868}
                value="Eritrea"
            >
                Eritrea
            </option>
            <option
                data-jsx={2272587868}
                value="Estonia"
            >
                Estonia
            </option>
            <option
                data-jsx={2272587868}
                value="Ethiopia"
            >
                Ethiopia
            </option>
            <option
                data-jsx={2272587868}
                value="Fiji"
            >
                Fiji
            </option>
            <option
                data-jsx={2272587868}
                value="Finland"
            >
                Finland
            </option>
            <option
                data-jsx={2272587868}
                value="France"
            >
                France
            </option>
            <option
                data-jsx={2272587868}
                value="Gabon"
            >
                Gabon
            </option>
            <option
                data-jsx={2272587868}
                value="Gambia"
            >
                Gambia
            </option>
            <option
                data-jsx={2272587868}
                value="Georgia"
            >
                Georgia
            </option>
            <option
                data-jsx={2272587868}
                value="Germany"
            >
                Germany
            </option>
            <option
                data-jsx={2272587868}
                value="Ghana"
            >
                Ghana
            </option>
            <option
                data-jsx={2272587868}
                value="Greece"
            >
                Greece
            </option>
            <option
                data-jsx={2272587868}
                value="Grenada"
            >
                Grenada
            </option>
            <option
                data-jsx={2272587868}
                value="Guatemala"
            >
                Guatemala
            </option>
            <option
                data-jsx={2272587868}
                value="Guinea Conakry"
            >
                Guinea Conakry
            </option>
            <option
                data-jsx={2272587868}
                value="Guinea-Bissau"
            >
                Guinea-Bissau
            </option>
            <option
                data-jsx={2272587868}
                value="Guyana"
            >
                Guyana
            </option>
            <option
                data-jsx={2272587868}
                value="Haiti"
            >
                Haiti
            </option>
            <option
                data-jsx={2272587868}
                value="Honduras"
            >
                Honduras
            </option>
            <option
                data-jsx={2272587868}
                value="Hungary"
            >
                Hungary
            </option>
            <option
                data-jsx={2272587868}
                value="Iceland"
            >
                Iceland
            </option>
            <option
                data-jsx={2272587868}
                value="India"
            >
                India
            </option>
            <option
                data-jsx={2272587868}
                value="Indonesia"
            >
                Indonesia
            </option>
            <option
                data-jsx={2272587868}
                value="Iran"
            >
                Iran
            </option>
            <option
                data-jsx={2272587868}
                value="Iraq"
            >
                Iraq
            </option>
            <option
                data-jsx={2272587868}
                value="Ireland"
            >
                Ireland
            </option>
            <option
                data-jsx={2272587868}
                value="Israel"
            >
                Israel
            </option>
            <option
                data-jsx={2272587868}
                value="Italy"
            >
                Italy
            </option>
            <option
                data-jsx={2272587868}
                value="Ivory Coast"
            >
                Ivory Coast
            </option>
            <option
                data-jsx={2272587868}
                value="Jamaica"
            >
                Jamaica
            </option>
            <option
                data-jsx={2272587868}
                value="Japan"
            >
                Japan
            </option>
            <option
                data-jsx={2272587868}
                value="Jordan"
            >
                Jordan
            </option>
            <option
                data-jsx={2272587868}
                value="Kazakhstan"
            >
                Kazakhstan
            </option>
            <option
                data-jsx={2272587868}
                value="Kenya"
            >
                Kenya
            </option>
            <option
                data-jsx={2272587868}
                value="Kiribati"
            >
                Kiribati
            </option>
            <option
                data-jsx={2272587868}
                value="Kosovo"
            >
                Kosovo
            </option>
            <option
                data-jsx={2272587868}
                value="Kuwait"
            >
                Kuwait
            </option>
            <option
                data-jsx={2272587868}
                value="Kyrgyzstan"
            >
                Kyrgyzstan
            </option>
            <option
                data-jsx={2272587868}
                value="Laos"
            >
                Laos
            </option>
            <option
                data-jsx={2272587868}
                value="Latvia"
            >
                Latvia
            </option>
            <option
                data-jsx={2272587868}
                value="Lebanon"
            >
                Lebanon
            </option>
            <option
                data-jsx={2272587868}
                value="Lesotho"
            >
                Lesotho
            </option>
            <option
                data-jsx={2272587868}
                value="Liberia"
            >
                Liberia
            </option>
            <option
                data-jsx={2272587868}
                value="Libya"
            >
                Libya
            </option>
            <option
                data-jsx={2272587868}
                value="Liechtenstein"
            >
                Liechtenstein
            </option>
            <option
                data-jsx={2272587868}
                value="Lithuania"
            >
                Lithuania
            </option>
            <option
                data-jsx={2272587868}
                value="Luxembourg"
            >
                Luxembourg
            </option>
            <option
                data-jsx={2272587868}
                value="Macedonia"
            >
                Macedonia
            </option>
            <option
                data-jsx={2272587868}
                value="Madagascar"
            >
                Madagascar
            </option>
            <option
                data-jsx={2272587868}
                value="Malawi"
            >
                Malawi
            </option>
            <option
                data-jsx={2272587868}
                value="Malaysia"
            >
                Malaysia
            </option>
            <option
                data-jsx={2272587868}
                value="Maldives"
            >
                Maldives
            </option>
            <option
                data-jsx={2272587868}
                value="Mali"
            >
                Mali
            </option>
            <option
                data-jsx={2272587868}
                value="Malta"
            >
                Malta
            </option>
            <option
                data-jsx={2272587868}
                value="Marshall Islands"
            >
                Marshall Islands
            </option>
            <option
                data-jsx={2272587868}
                value="Mauritania"
            >
                Mauritania
            </option>
            <option
                data-jsx={2272587868}
                value="Mauritius"
            >
                Mauritius
            </option>
            <option
                data-jsx={2272587868}
                value="Mexico"
            >
                Mexico
            </option>
            <option
                data-jsx={2272587868}
                value="Micronesia"
            >
                Micronesia
            </option>
            <option
                data-jsx={2272587868}
                value="Moldova"
            >
                Moldova
            </option>
            <option
                data-jsx={2272587868}
                value="Monaco"
            >
                Monaco
            </option>
            <option
                data-jsx={2272587868}
                value="Mongolia"
            >
                Mongolia
            </option>
            <option
                data-jsx={2272587868}
                value="Montenegro"
            >
                Montenegro
            </option>
            <option
                data-jsx={2272587868}
                value="Morocco"
            >
                Morocco
            </option>
            <option
                data-jsx={2272587868}
                value="Mozambique"
            >
                Mozambique
            </option>
            <option
                data-jsx={2272587868}
                value="Myanmar [Burma]"
            >
                Myanmar [Burma]
            </option>
            <option
                data-jsx={2272587868}
                value="Namibia"
            >
                Namibia
            </option>
            <option
                data-jsx={2272587868}
                value="Nauru"
            >
                Nauru
            </option>
            <option
                data-jsx={2272587868}
                value="Nepal"
            >
                Nepal
            </option>
            <option
                data-jsx={2272587868}
                value="Netherlands"
            >
                Netherlands
            </option>
            <option
                data-jsx={2272587868}
                value="New Caledonia"
            >
                New Caledonia
            </option>
            <option
                data-jsx={2272587868}
                value="New Zealand"
            >
                New Zealand
            </option>
            <option
                data-jsx={2272587868}
                value="Nicaragua"
            >
                Nicaragua
            </option>
            <option
                data-jsx={2272587868}
                value="Niger"
            >
                Niger
            </option>
            <option
                data-jsx={2272587868}
                value="Nigeria"
            >
                Nigeria
            </option>
            <option
                data-jsx={2272587868}
                value="North Korea"
            >
                North Korea
            </option>
            <option
                data-jsx={2272587868}
                value="Norway"
            >
                Norway
            </option>
            <option
                data-jsx={2272587868}
                value="Oman"
            >
                Oman
            </option>
            <option
                data-jsx={2272587868}
                value="Pakistan"
            >
                Pakistan
            </option>
            <option
                data-jsx={2272587868}
                value="Palau"
            >
                Palau
            </option>
            <option
                data-jsx={2272587868}
                value="Palestine"
            >
                Palestine
            </option>
            <option
                data-jsx={2272587868}
                value="Panama"
            >
                Panama
            </option>
            <option
                data-jsx={2272587868}
                value="Paraguay"
            >
                Paraguay
            </option>
            <option
                data-jsx={2272587868}
                value="Peru"
            >
                Peru
            </option>
            <option
                data-jsx={2272587868}
                value="Philippines"
            >
                Philippines
            </option>
            <option
                data-jsx={2272587868}
                value="Poland"
            >
                Poland
            </option>
            <option
                data-jsx={2272587868}
                value="Portugal"
            >
                Portugal
            </option>
            <option
                data-jsx={2272587868}
                value="Qatar"
            >
                Qatar
            </option>
            <option
                data-jsx={2272587868}
                value="Republic of Korea"
            >
                Republic of Korea
            </option>
            <option
                data-jsx={2272587868}
                value="Republic of the Congo"
            >
                Republic of the Congo
            </option>
            <option
                data-jsx={2272587868}
                value="Romania"
            >
                Romania
            </option>
            <option
                data-jsx={2272587868}
                value="Russia"
            >
                Russia
            </option>
            <option
                data-jsx={2272587868}
                value="Rwanda"
            >
                Rwanda
            </option>
            <option
                data-jsx={2272587868}
                value="Saint Kitts and Nevis"
            >
                Saint Kitts and Nevis
            </option>
            <option
                data-jsx={2272587868}
                value="Saint Lucia"
            >
                Saint Lucia
            </option>
            <option
                data-jsx={2272587868}
                value="Saint Vincent and the Grenadines"
            >
                Saint Vincent and the Grenadines
            </option>
            <option
                data-jsx={2272587868}
                value="Samoa"
            >
                Samoa
            </option>
            <option
                data-jsx={2272587868}
                value="San Marino"
            >
                San Marino
            </option>
            <option
                data-jsx={2272587868}
                value="Saudi Arabia"
            >
                Saudi Arabia
            </option>
            <option
                data-jsx={2272587868}
                value="Senegal"
            >
                Senegal
            </option>
            <option
                data-jsx={2272587868}
                value="Serbia"
            >
                Serbia
            </option>
            <option
                data-jsx={2272587868}
                value="Seychelles"
            >
                Seychelles
            </option>
            <option
                data-jsx={2272587868}
                value="Sierra Leone"
            >
                Sierra Leone
            </option>
            <option
                data-jsx={2272587868}
                value="Singapore"
            >
                Singapore
            </option>
            <option
                data-jsx={2272587868}
                value="Slovak Republic"
            >
                Slovak Republic
            </option>
            <option
                data-jsx={2272587868}
                value="Slovenia"
            >
                Slovenia
            </option>
            <option
                data-jsx={2272587868}
                value="Solomon Islands"
            >
                Solomon Islands
            </option>
            <option
                data-jsx={2272587868}
                value="Somalia"
            >
                Somalia
            </option>
            <option
                data-jsx={2272587868}
                value="South Africa"
            >
                South Africa
            </option>
            <option
                data-jsx={2272587868}
                value="South Sudan"
            >
                South Sudan
            </option>
            <option
                data-jsx={2272587868}
                value="Spain"
            >
                Spain
            </option>
            <option
                data-jsx={2272587868}
                value="Sri Lanka"
            >
                Sri Lanka
            </option>
            <option
                data-jsx={2272587868}
                value="Sudan"
            >
                Sudan
            </option>
            <option
                data-jsx={2272587868}
                value="Suriname"
            >
                Suriname
            </option>
            <option
                data-jsx={2272587868}
                value="Swaziland"
            >
                Swaziland
            </option>
            <option
                data-jsx={2272587868}
                value="Sweden"
            >
                Sweden
            </option>
            <option
                data-jsx={2272587868}
                value="Switzerland"
            >
                Switzerland
            </option>
            <option
                data-jsx={2272587868}
                value="Syria"
            >
                Syria
            </option>
            <option
                data-jsx={2272587868}
                value="São Tomé and Príncipe"
            >
                São Tomé and Príncipe
            </option>
            <option
                data-jsx={2272587868}
                value="Taiwan"
            >
                Taiwan
            </option>
            <option
                data-jsx={2272587868}
                value="Tajikistan"
            >
                Tajikistan
            </option>
            <option
                data-jsx={2272587868}
                value="Tanzania"
            >
                Tanzania
            </option>
            <option
                data-jsx={2272587868}
                value="Thailand"
            >
                Thailand
            </option>
            <option
                data-jsx={2272587868}
                value="Togo"
            >
                Togo
            </option>
            <option
                data-jsx={2272587868}
                value="Tonga"
            >
                Tonga
            </option>
            <option
                data-jsx={2272587868}
                value="Trinidad and Tobago"
            >
                Trinidad and Tobago
            </option>
            <option
                data-jsx={2272587868}
                value="Tunisia"
            >
                Tunisia
            </option>
            <option
                data-jsx={2272587868}
                value="Turkey"
            >
                Turkey
            </option>
            <option
                data-jsx={2272587868}
                value="Turkmenistan"
            >
                Turkmenistan
            </option>
            <option
                data-jsx={2272587868}
                value="Tuvalu"
            >
                Tuvalu
            </option>
            <option
                data-jsx={2272587868}
                value="Uganda"
            >
                Uganda
            </option>
            <option
                data-jsx={2272587868}
                value="Ukraine"
            >
                Ukraine
            </option>
            <option
                data-jsx={2272587868}
                value="United Arab Emirates"
            >
                United Arab Emirates
            </option>
            <option
                data-jsx={2272587868}
                value="United Kingdom"
            >
                United Kingdom
            </option>
            <option
                data-jsx={2272587868}
                value="United States"
            >
                United States
            </option>
            <option
                data-jsx={2272587868}
                value="Uruguay"
            >
                Uruguay
            </option>
            <option
                data-jsx={2272587868}
                value="Uzbekistan"
            >
                Uzbekistan
            </option>
            <option
                data-jsx={2272587868}
                value="Vanuatu"
            >
                Vanuatu
            </option>
            <option
                data-jsx={2272587868}
                value="Venezuela"
            >
                Venezuela
            </option>
            <option
                data-jsx={2272587868}
                value="Vietnam"
            >
                Vietnam
            </option>
            <option
                data-jsx={2272587868}
                value="Yemen"
            >
                Yemen
            </option>
            <option
                data-jsx={2272587868}
                value="Zambia"
            >
                Zambia
            </option>
            <option
                data-jsx={2272587868}
                value="Zimbabwe"
            >
                Zimbabwe
            </option>
        </select>
    </div>
    <button
        className="btn"
        data-jsx={2272587868}
        disabled={false}
        onClick={[Function]}
        style={
            Object {
                "background": "",
                "color": "#fff",
                "marginLeft": "-2px",
              }
        }
    >
         
        
    </button>
    <div
        className="input-container"
        data-jsx={2272587868}
    >
        <div
            className="hidden"
            data-jsx={2272587868}
        />
    </div>
    <span
        className="message"
        data-jsx={2272587868}
        style={
            Object {
                "display": "none",
              }
        }
    >
        you're already praying
    </span>
    <_class
        css=".message[data-jsx=\\"2272587868\\"]{color:#fff, width:90%, padding:10px, margin:5px auto, background:#f4334a,}"
        styleId={2272587868}
    />
</form>,
]
`;
